src/patterns/start-pages/index.md.njk

```diff
@@ -0,0 +1,53 @@
+---
+title: Start pages
+description:
+section: Patterns
+theme: Pages
+aliases:
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+If you’re building a digital service, you’ll need to provide a starting point for users on GOV.UK.
+
+{{ example({group: 'patterns', item: 'start-pages', example: 'default', html: true, nunjucks: true, open: true}) }}
+
+## When to use this pattern
+
+If your service is in public beta or live, you must have a starting point hosted on GOV.UK. This will link your service to the rest of GOV.UK.
+
+## How it works
+
+Start pages include 4 main elements:
+
+1. The service name: this should help people understand what your service does and whether they need to use it - [learn more about naming your service](https://www.gov.uk/service-manual/design/naming-your-service).
+
+2. A short introduction to list things that most users will need to know, for example, what your service will let them do or how much it’ll cost.
+
+3. The ‘Start now’ call-to-action button.
+
+4. A list of other ways to access your service.
+
+This is a set pattern, so you won’t be able to customise it. 
+
+
+### Testing start pages in discovery and alpha
+
+If you want to prototype designs so you can test them with users, you can use the [start page template](https://govuk-prototype-kit-beta.herokuapp.com/docs/examples/start-page) in the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com/).
+
+### Get a start page for your public beta or live service
+
+[Contact the GDS content team](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk) before your alpha assessment to discuss start points for your service. 
+
+The GDS content team will create a start page for you in most cases. If this isn’t the best way to meet the user need, they’ll recommend another way of joining up with the rest of GOV.UK.
+
+GDS will only publish your start page when your service:
+
+- passes its beta assessment or self-certification
+- goes into public beta
```

Comment:
this could be read as an 'or' list, and is only two items - maybe clearer as a sentence or two?

Reply:
Fine - fixed.